src/solve/canonicalization.md

```diff
@@ -0,0 +1,8 @@
+# Canonicalization
+
+While the exact approach to canonicalization for this solver will differ slightly
+wrt to lifetimes, please visit [the relevant chalk chapter][chalk] for now.
+
+As of the 10th January 2023 canonicalization is not yet fully implemented for the solver.
```

Comment:
```suggestion
As of 10<!-- date-check --> January 2023, canonicalization is not yet fully implemented for the solver.
```
We have a tool that checks these hidden comments to ensure that we get notified if things get stale (currently set at 6 months), and it produces [this example report](https://github.com/rust-lang/rustc-dev-guide/issues/1539).